docs/howto/getting-started/accessing-cc-rest-api.md

```diff
@@ -0,0 +1,138 @@
+# Accessing the Cleura Cloud REST API
+
+{{brand}} provides a REST API, which you can take advantage of with a 
+tool like `curl`. But before you do, you must create a token for the 
+current session. For that, you only need to have an
+[account in {{brand}}](/howto/getting-started/create-account).
+
+## Creating a token
+
+To create a token, have your username (`your_cc_username`) and password 
+(`your_cc_password`) to the {{brand}} handy, and type:
+
+```bash
+curl -H "Content-Type: application/json" -X POST -d \
+    '{"auth": {"login": "your_cc_username", "password": "your_cc_password"}}' \
+    https://rest.cleura.cloud/auth/v1/tokens
```

Comment:
Setting `-d` already makes `curl` send a `POST` request, `-X POST` is thus redundant.

```suggestion
curl -H "Content-Type: application/json" \
    -d '{"auth": {"login": "your_cc_username", "password": "your_cc_password"}}' \
    https://rest.cleura.cloud/auth/v1/tokens
```